docs/source/conf.py

```diff
@@ -14,7 +14,11 @@
 # They can be extensions coming with Sphinx (named 'sphinx.ext.*')
 # or your custom ones.
 extensions = [
-    'sphinx.ext.autodoc', 'sphinx.ext.githubpages', 'sphinx.ext.napoleon']
+    'sphinx.ext.autodoc',
+    'sphinx.ext.githubpages',
+    'sphinx.ext.napoleon'
+    ]
+
```

Comment:
This does not match [the style guideline](https://mcsinyx.github.io/palace/contributing.html#style-guidelines), please revert to the original one.